imap_processing/ultra/tests/plugins/data_path_fixtures.py

```diff
@@ -0,0 +1,62 @@
+"""Pytest plugin module for test data paths"""
+# Standard
+import sys
+from pathlib import Path
+
+# Installed
+import pytest
+
+from imap_processing import packet_definition_directory
+
+
+@pytest.fixture()
+def test_data_path():
+    """Returns the Path to the test_data directory"""
+    return Path(sys.modules[__name__.split(
+        '.')[0]].__file__).parent / 'ultra'/ 'tests' / 'test_data'
```

Comment:
```suggestion
    return Path(__file__).parent.parent / 'ultra'/ 'tests' / 'test_data'
```
But note you'd only need one parent if you move this over to conftest.